### ISSUE TITLE
CSS is broken

### ISSUE THREAD

**Issue by pragma-** [plfa/plfa.github.io#706]
The CSS on the github.io site has been broken for a couple of days so I've been using https://plfa.inf.ed.ac.uk/Induction/ instead. Apparently that has now updated and the CSS is now broken there as well.

**Comment by wenkokke** [comment]
The CSS has been broken since roundabout yesterday evening. It has now been fixed on https://plfa.github.io/. The inf.ed.ac.uk is a nightly build and as such will remain broken for another 18 or so hours.

**Comment by pragma-** [comment]
I'm glad the issue is now resolved. I am enjoying reading this book.

But you said "since yesterday evening"; however, I first noticed the broken CSS approximately 4 days ago coinciding with this commit https://github.com/plfa/plfa.github.io/commit/42fe52c01a9ae3b26b4bdb87e9f301b6d9f8cf03. I had then immediately switched to using the pfla.inf.ed.ac.uk site, which continued to render correctly for the next 4 days until today.

I apologize for this seemingly pointless comment on a closed issue but I felt the need to defend the data I provided when I opened the issue.
